deepchem/package.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Can you add a line to README explaining how package-lock.json is updated?

Reply:
Added necessary documentation.